README.md

```diff
@@ -1,2 +1,110 @@
-# Budget
-Budget-app is a mobile web application that assists users with money management. It may also show consumers a list of transactions they've done that are tied to their categories, as well as the overall amount of money they've spent.
+![Microverse](https://img.shields.io/badge/-Microverse-%20%236f23ff?style=for-the-badge) 
+
+# Budget-app
+
+> Budget-app is a mobile web application that assists users with money management. It may also show consumers a list of transactions they've done that are tied to their categories, as well as the overall amount of money they've spent.
+
+## Live Demo
+
+[Live Demo Link](https://agile-woodland-22089.herokuapp.com/)
+
+## Video Link
+
+[VIDEO LINK](https://www.loom.com/share/a4f8625f482f455a81ade10c0f43ac88)
+
```

Comment:
- [ ] In your video you should highlight a piece of the code that you find interesting or you are proud of making.